mkosi/distributions/fedora.py

```diff
@@ -29,18 +30,20 @@ def find_fedora_rpm_gpgkeys(context: Context) -> Iterable[str]:
         context, key=f"RPM-GPG-KEY-fedora-{context.config.release}-secondary", required=False
     )
 
+    versionre = r"RPM-GPG-KEY-fedora-(\d+)-(primary|secondary)"
```

Comment:
```suggestion
    versionre = re.compile(r"RPM-GPG-KEY-fedora-(\d+)-(primary|secondary)")
```
so that below you can do, e.g. 
```
versionre.match(Path(rhs).name)
```

Reply:
You could also do
```
versionre = re.compile(r"RPM-GPG-KEY-fedora-(?P<versionnum>\d+)-(primary|secondary)")
```
so that the group doesn't have to be looked up by number